### ISSUE TITLE
Project state?

### ISSUE THREAD

**Issue by Arbee4ever** [Harium/supabase-java#6]
Hey, is this Project in a usable State or has it been abandoned? I could try fork and update it...

**Comment by yuripourre** [comment]
Hey, @Arbee4ever, this project works perfectly.

I am not currently working on it but feel free to open PRs or fork it (either works for me). Please let me know how I can help. I can't remember if it's published on maven but if you need it just tell me.

**Comment by Arbee4ever** [comment]
Well I'm using `implementation 'com.github.Harium:supabase-java:ac0f109'` in my build.gradle right now, because the 0.0.1 release on maven didn't work... Also it doesn't seem like I can use different Schemas can I?

**Comment by Arbee4ever** [comment]
I'm using that specific commit, because the one after updates a dependency to 0.0.3 which doesn't exist on maven (only 0.0.1 and 0.0.2 exist)

**Comment by yuripourre** [comment]
> Well I'm using `implementation 'com.github.Harium:supabase-java:ac0f109'` in my build.gradle right now, because the 0.0.1 release on maven didn't work... Also it doesn't seem like I can use different Schemas can I?

Would you mind elaborate what you mean by different schemas?

> I'm using that specific commit, because the one after updates a dependency to 0.0.3 which doesn't exist on maven (only 0.0.1 and 0.0.2 exist)

Ops, sorry about that, I probably didn't publish the latest version, tbh I had no idea someone other than myself would use it. I can see if I publish later this week.

**Comment by Arbee4ever** [comment]
With Schemas I mean this: https://supabase.com/docs/guides/database/tables#schemas

**Comment by yuripourre** [comment]
> With Schemas I mean this: https://supabase.com/docs/guides/database/tables#schemas

Oh, right, you mean database schema. You can create any kind of schema you just need to make sure that the calls are consistent.

```
supabase.databse().insert("mytable", Insert.row().column("column1", "mydata")
                                     .column("column2", "1234"));
supabase.databse().insert("mytable", Insert.row().column("column1", "more-data")
                                     .column("column2", "12345"));
```

**Comment by Arbee4ever** [comment]
How do I tell it which Schema to use though?

**Comment by yuripourre** [comment]
> How do I tell it which Schema to use though?

Sorry, maybe I didn't understand your question. You can define as many columns as you want with the syntax I sent earlier. Would you mind giving me an example of schema you are trying to create?

**Comment by Arbee4ever** [comment]
I don't think you really understand what I mean with schemas :smile: The schemas I mean group multiple tables... they don't do anything with columns

**Comment by yuripourre** [comment]
> I don't think you really understand what I mean with schemas 😄 The schemas I mean group multiple tables... they don't do anything with columns

Oh! I see what you mean now. I am not a database expert myself. I created the [postgrest-java](https://github.com/Harium/postgrest-java) library based on the link below.

Can you find what you need here? https://postgrest.org/en/stable/references/api/tables_views.html

**Comment by Arbee4ever** [comment]
It should be [this](https://postgrest.org/en/stable/references/api/schemas.html#get-head)

**Comment by yuripourre** [comment]
> It should be [this](https://postgrest.org/en/stable/references/api/schemas.html#get-head)

Right, I get it now. Yeah, it's not implemented. I will create a issue to track this request but I have no time to work on this. If you have time, please feel free to fork or open a PR, otherwise it might take a while.

**Comment by yuripourre** [comment]
@Arbee4ever issue created: https://github.com/Harium/postgrest-java/issues/1

**Comment by Arbee4ever** [comment]
> @Arbee4ever issue created: https://github.com/Harium/postgrest-java/issues/1

Thanks... I'll see what I can do

**Comment by Arbee4ever** [comment]
I don't know how you'd do that but maybe you can add this library to supabase's Docs? Just like [supabase-kt](https://supabase.com/docs/reference/kotlin/introduction)

**Comment by yuripourre** [comment]
> I don't know how you'd do that but maybe you can add this library to supabase's Docs? Just like [supabase-kt](https://supabase.com/docs/reference/kotlin/introduction)

Thanks for the suggestion, I created a discussion: https://github.com/orgs/supabase/discussions/17958
